lib/config/httpRoutes.js

```diff
@@ -287,6 +287,8 @@ const routes = [
   { verb: 'delete', path: '/:index/:collection/_mDelete', controller: 'document', action: 'mDelete' },
 
   { verb: 'delete', path: '/:index', controller: 'index', action: 'delete' },
+  { verb: 'delete', path: '/_mDelete', controller: 'index', action: 'mDelete' },
+  // @deprecated: has a typo. To be removed in the next major version
   { verb: 'delete', path: '/_mdelete', controller: 'index', action: 'mDelete' },
```

Comment:
Since this is deprecated, can we add a deprecation notice ?

Reply:
ah, sure, I forgot about that new feature :+1:

Reply:
Done :+1: